File: RECURSION/GCD.c
#include <stdio.h>
int GCD(int a, int b)
{
    if (b == 0)
        return a;
    else
        return GCD(b, a % b);
}

int main()
{
    int a, b, result;

    printf("Enter the value of a ,b");
    scanf("%d%d", &a, &b);

    result = GCD(a, b);
    printf("The greates common divisor of %d and %d is : %d ", a, b, result);

    return 0;
}

File: RECURSION/GCD.py
def GCD(a, b):
    if b == 0:
        return a
    else:
        return GCD(b, a % b)


a, b = map(int, input("Enter two numbers separated by spaces: ").split())
result = GCD(a, b)
print(f"The greatest common divisor between {a} and {b} is {result}")
